ali/commands/template.py

```diff
@@ -0,0 +1,64 @@
+import json
```

Comment:
Should we put this file into the `ali.helpers` package instead of `ali.commands`? The module doesn't add any new commands to the CLI, so I guess that would make more sense.